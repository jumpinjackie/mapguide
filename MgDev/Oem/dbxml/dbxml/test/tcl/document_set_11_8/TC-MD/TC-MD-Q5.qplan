<XQuery>
  <FLWOR>
    <ForBinding name="a">
      <QueryPlanToAST>
        <NodePredicateFilterQP uri="" name="#tmp1">
          <StepQP axis="child" name="article" nodeType="element">
            <ValueQP index="node-attribute-equality-string" operation="eq" child="id" value="9"/>
          </StepQP>
          <ValueFilterQP comparison="eq" general="true">
            <StepQP axis="attribute" name="id" nodeType="attribute">
              <VariableQP name="#tmp1"/>
            </StepQP>
            <Sequence>
              <AnyAtomicTypeConstructor value="9" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
            </Sequence>
          </ValueFilterQP>
        </NodePredicateFilterQP>
      </QueryPlanToAST>
    </ForBinding>
    <DOMConstructor type="element">
      <Name>
        <Sequence>
          <ATQNameConstructor uri="" prefix="" localname="HeadingOfSection" typeuri="http://www.w3.org/2001/XMLSchema" typename="QName"/>
        </Sequence>
      </Name>
      <Children>
        <ContentSequence>
          <QueryPlanToAST>
            <StepQP axis="attribute" name="heading" nodeType="attribute">
              <ASTToQueryPlan>
                <Map>
                  <QueryPlanToAST>
                    <StepQP axis="child" name="body" nodeType="element">
                      <VariableQP name=":a"/>
                    </StepQP>
                  </QueryPlanToAST>
                  <QueryPlanToAST>
                    <NumericPredicateFilterQP>
                      <StepQP axis="child" name="section" nodeType="element">
                        <ContextNodeQP/>
                      </StepQP>
                      <Sequence>
                        <NumericTypeConstructor value="1.0E0" typeuri="http://www.w3.org/2001/XMLSchema" typename="integer"/>
                      </Sequence>
                    </NumericPredicateFilterQP>
                  </QueryPlanToAST>
                </Map>
              </ASTToQueryPlan>
            </StepQP>
          </QueryPlanToAST>
        </ContentSequence>
      </Children>
    </DOMConstructor>
  </FLWOR>
</XQuery>
