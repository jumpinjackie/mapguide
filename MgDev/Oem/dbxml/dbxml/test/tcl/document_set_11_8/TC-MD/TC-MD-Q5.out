<HeadingOfSection heading="sheaves beneath the ironic, permanent realms should have to use upon "/>